calendars/community.yaml

```
@@ -10,31 +10,36 @@ events:
   - &ccall
     summary: CodeRefinery Community call
     location: https://hackmd.io/@coderefinery/community-call
-    begin: 2022-01-31 14:00:00
+    begin: 2022-01-09 14:00:00
```

```suggestion
    begin: 2023-01-09 14:00:00
```
Oh yes, at first I did it a different way and forgot to update this.  very good catch!